### ISSUE TITLE
Inference question

### ISSUE THREAD

**Issue by qiqiApink** [fmthoker/skeleton-contrast#10]
How to inference this model on other human skeleton data, like human3.6m? I want to classify the action categories of skeleton sequences. Does this project achieve this?

**Comment by fmthoker** [comment]
you need to first pre-train a model with your dataset without any action labels. Then, you can fine-tune a model for action classification.
